### DISCUSSION TITLE
set a custom startup animation for android apk

### DISCUSSION THREAD

**Discussion by Jzhenli** [beeware/briefcase#1945] [Q&A]
Hi, the apk startup to slow, so I want to set a custom startup animation, but don't know how to do it with briefcase,  would u give a brief example?

**Comment by freakboy3742** [comment]
At present, Briefcase provides a standard Android 12 [splash screen](https://developer.android.com/develop/ui/views/launch/splash-screen) based on an [adaptive icon](https://developer.android.com/develop/ui/views/launch/icon_design_adaptive).

You could customise your app to add animated splash screen as described in the Android docs; to do this, you would use the Briefcase configuration items like [`build_grade_dependencies`](https://briefcase.readthedocs.io/en/stable/reference/platforms/android/gradle.html#build-gradle-dependencies) and [`android_manifest_activity_extra_content`](https://briefcase.readthedocs.io/en/stable/reference/platforms/android/gradle.html#android-manifest-activity-extra-content) to inject extra content into your AndroidManifest.xml and build.gradle files.
